ormconfig.travis.json

```diff
@@ -36,6 +36,11 @@
     "database": "test"
   },
```

Comment:
Also add sqljs to ormconfig template for other developers `ormconfig.json.dist`

Reply:
I added it